src/bin/lttng-relayd/main.c

```diff
@@ -3403,8 +3403,11 @@ static void *relay_thread_worker(void *data)
 					if (ret < 0) {
 						goto error;
 					}
-					lttng_poll_add(&events, conn->sock->fd,
+					ret = lttng_poll_add(&events, conn->sock->fd,
```

Comment:
No reason this wasn't checked :) thanks!
I added a logging statement in both cases.